test/known_issues/test-url-parse-conformance.js

```diff
@@ -1,13 +1,11 @@
+/* eslint-disable required-modules */
```

Comment:
I had to add this because the common package is not used after this change and therefore it conflicted with`no-unused-vars`